docs/lcdproc-user/how-to-obtain.docbook

```diff
@@ -25,8 +25,8 @@ around on the internet:
   <term>Older releases</term>
   <listitem>
     <para>
-    Older releases of LCDproc can be found on the Sourceforge project page at
-    <ulink url="http://sourceforge.net/projects/lcdproc/files/lcdproc/"></ulink>.
+    Older releases of LCDproc can be found on the GitHub project page at
+    <ulink url="https://github.com/lcdproc/lcdproc"></ulink>.
     </para>
```

Comment:
This is not true at the moment. I'm not sure what should be written here instead, though.